README.md

```diff
@@ -82,39 +81,11 @@ We plan to utilize API versioning to make automatic updates safe for consumer co
 
 To use the Block Data API after activation, skip to [Usage](#usage).
 
-### Install via `git subtree`
```

Comment:
@ingeniumed Are we deleting this section? I think it's still useful for VIP customers uninterested in the automatic integration mode, and non-VIP customers that want to add the plugin to existing repositories.

Reply:
Discussed it, and will keep this change.